### DISCUSSION TITLE
Out-of-the-box Remix app not working

### DISCUSSION THREAD

**Discussion by SantiSnow** [remix-run/remix#1935] [Q&A] [answered]
Hi guys. I was trying Remix for the first time, but following the steps in the Docs it fails at `npm start`

### Steps to reproduce the bug:

- npx create-remix@latest

- Where would you like to create your app? remix-app
- Where do you want to deploy? Choose Remix if you're unsure, it's easy to change deployment targets. Remix
- App Server
- TypeScript or JavaScript? TypeScript
- Do you want me to run `npm install`? Yes

- cd remix-app
- code .
- npm run build
- npm start

### Result:
`
> remix-serve build

internal/modules/cjs/loader.js:818
  throw err;
  ^

Error: Cannot find module 'fs/promises'
Require stack:
- /home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/node/errors.js
- /home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/node/index.js
- /home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/express/globals.js
- /home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/express/index.js
- /home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/serve/index.js
- /home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/serve/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/node/errors.js:16:11)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/node/errors.js',
    '/home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/node/index.js',
    '/home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/express/globals.js',
    '/home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/express/index.js',
    '/home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/serve/index.js',
    '/home/santiago/Documents/Trabajo/Personal/my-remix-app/node_modules/@remix-run/serve/cli.js'
  ]
}
`

This was tested in 3 diferent computers, Windows, MacOS and Ubuntu 20. Others npm/node related technologies work fine in my computers so I am pretty sure it's not related with my npm/node installation.

**Comment by MichaelDeBoey** [comment, marked as answer]
As described in https://github.com/remix-run/remix/issues/1936#issuecomment-1038226911, this works locally on Node 14 for me
